src/game-logic/preGame.js

```diff
@@ -0,0 +1,61 @@
+import cardRoundLogic from './card-logic/cardRoundLogic';
+
+/**
+ * Pre-game logic that occurs before the spell-card game begins
+ */
+const TIMERLENGTH = 30000;
+
+export default function pregame(socket, roomID) {
+   // start timer in socket.io room
+   socket.to(roomID).emit('start pre-game');
+
+   // send first 5 cards to each user in the room
+   // TODO: turn into playerID 
+   // cards selected, and send cards to game
+   let cards = cardRoundLogic();
+   socket.to(roomID).emit('cards', cards)
+
+   // On either timer TimeOut or Continue button press, start game
+   let preGamePromsie = new Promise((fulfill, reject) => {
```

Comment:
What is the reason for writing this as a promise rather than an async function?

Reply:
no reasoning, just what I decided to do.

Reply:
IMO would be cleaner/easier to understand as a function